frontend/components/basic/TabBar.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <TabBarScoob v-if="showScoob"></TabBarScoob>
+  <TabBarMagAOX v-else></TabBarMagAOX>
+</template>
+<script>
+import constants from "~/constants.js";
+import TabBarMagAOX from "~/pages/MagAOX/TabBar.vue";
+import TabBarScoob from "~/pages/Scoob/TabBar.vue";
+
+export default {
+  components: {
+    TabBarMagAOX,
+    TabBarScoob,
+  },
+  computed: {
+    showScoob: function () {
```

Comment:
Since we're refactoring, we might as well future-proof for the Nth instrument to use this UI... instead of a boolean yes-scoob/no-scoob we should test for equality in the `v-if` in the template and have a fallback for if an unknown layout is requested that says something useful (to catch misconfigurations). Then the computed property won't need to be renamed when support for a third instrument is added

Reply:
Very good point, I've made the change.